src/ui/components/Timeline/FocusModePopout.tsx

```diff
@@ -15,6 +17,8 @@ import Icon from "../shared/Icon";
 import styles from "./FocusModePopout.module.css";
 
 export default function FocusModePopout() {
+  const [useFocusModeState, dismissUseFocusModeNag] = useNag(Nag.USE_FOCUS_MODE);
```

Comment:
nit: For readability
```suggestion
  const [, dismissUseFocusModeNag] = useNag(Nag.USE_FOCUS_MODE);
```